README.md

```diff
@@ -17,7 +21,7 @@ python3 -m venv venv
 # In Windows PowerShell run this instead: `.\venv\Scripts\Activate.ps1`
 # In Windows CMD run this instead: `.\venv\scripts\activate.bat`
 source venv/bin/activate
-pip install --upgrade pip
+pip install --upgrade pip tox
```

Comment:
Was there a change that made this necessary? `tox` is still installed via the command below, isn't it?